src/main/java/seedu/duke/common/Messages.java

```diff
@@ -0,0 +1,31 @@
+package seedu.duke.common;
+
+import seedu.duke.ui.Colors;
+
+public class Messages {
+    // Application name
+    public static final String APPLICATION_NAME = Colors.ANSI_YELLOW + "MediTask" + Colors.ANSI_RESET;
+
+    // Logo
+    public static final String LOGO = """
+                  _____
+                 [IIIII]
+                 )"\"\""\"(
+                /       \\
+               /         \\
+               |`-.....-'|
+               | MediTask|
+             _ |`-.....-'|     _
+            (\\)`-.__.__.(I) _(/)
+              (I)  (/)(I)(\\)
+                 (I)        Task management for medical professionals
+                        """;;
+
+    // Messages
```

Comment:
Maybe we can further group them to prompts and welcome/goodbye so it is easier to manage down the line as we have more messages.